### ISSUE TITLE
Add ability to reset password

### ISSUE THREAD

**Issue by connorphee** [connorphee/league_app#2]
Reliant on #1, users will need to be able to reset password if it is forgotten.

**Comment by wildan3105** [comment]
Hi, I can work on this. Do you give me a shot? @connorphee

**Comment by connorphee** [comment]
@wildan3105 yes absolutely go ahead! As I have mentioned in other issues, I am working on docs currently, and will hopefully have them up by then end of the day!

If you have any issues please let me know!

**Comment by wildan3105** [comment]
So, my idea is to send a confirmation email contains unique/random password to user's email and also reset link, is that make sense? @connorphee

**Comment by connorphee** [comment]
@wildan3105 you may take this however you want. I imagined a reset email being sent, but any method works! Yours sounds great!

**Comment by wildan3105** [comment]
Oh, I see. Okay I'll do it my way :ok:

**Comment by connorphee** [comment]
Thank you very much! I appreciate the help!!

**Comment by wildan3105** [comment]
Hi, one last thing. I think you need to add the content about set up the project locally because I have an issue about that @connorphee. Thanks

**Comment by connorphee** [comment]
Yup I am doing that right now. Will comment back here when done

**Comment by connorphee** [comment]
@wildan3105 what is giving you trouble with the setup locally?

**Comment by wildan3105** [comment]
I can't find any configuration file that describe the connection to MongoDB.

**Comment by wildan3105** [comment]
Ah I see, forgot to read the `CONTRIBUTING.md` file. Should resolve that

**Comment by connorphee** [comment]
Awesome, I will try to make the more clear. If you need anything else please let me know I am nearby my computer for most of the day, outside of little time in class!

**Comment by Vishwajeetsinh98** [comment]
@connorphee do you have sendgrid or some other email sender setup? We'll need that to send emails using the app to reset the password.

**Comment by connorphee** [comment]
@Vishwajeetsinh98 I would check out #7, I believe @wildan3105 was able to handle the sender setup in his PR but the issue is in the creation and storing of a new password. He had mentioned using something like [simple encryptor](https://www.npmjs.com/package/simple-encryptor) for this.

**Comment by Vishwajeetsinh98** [comment]
@connorphee I've looked at the code. I'll fix the encryption thing and let you know.
